src/Payment/Aggregates/Payments/AbstractPayment.php

```diff
@@ -5,11 +5,13 @@
 use MundiAPILib\Models\CreatePaymentRequest;
 use Pagarme\Core\Kernel\Abstractions\AbstractEntity;
 use Pagarme\Core\Kernel\Abstractions\AbstractModuleCoreSetup as MPSetup;
+use Pagarme\Core\Marketplace\Aggregates\Split;
 use Pagarme\Core\Payment\Interfaces\ConvertibleToSDKRequestsInterface;
 use Pagarme\Core\Payment\Interfaces\HaveOrderInterface;
 use Pagarme\Core\Payment\Traits\WithAmountTrait;
 use Pagarme\Core\Payment\Traits\WithCustomerTrait;
 use Pagarme\Core\Payment\Traits\WithOrderTrait;
+use Pagarme\Pagarme\Concrete\Magento2CoreSetup;
```

Comment:
A gente não pode fazer o import de uma classe concreta do módulo de Magento 2 no core. Se for necessário usar algo do `Magento2CoreSetup`, deveriamos chamar o método pelo `AbstractModuleCoreSetup` do próprio core

Reply:
corrigido!